clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/deploy/ops/DeployAppengineAtomicOperation.groovy

```diff
@@ -53,21 +53,32 @@ class DeployAppengineAtomicOperation implements AtomicOperation<DeploymentResult
 
   DeployAppengineDescription description
   boolean usesGcs
+  boolean isContainerDeployment
```

Comment:
I'd skip the `is` here - the auto generated getters will add `is` automatically (`op.isGeneratedDeployment()` in non-groovy code).